### DISCUSSION TITLE
cuda 12.6

### DISCUSSION THREAD

**Discussion by eljandoubi** [bitsandbytes-foundation/bitsandbytes#1399] [Q&A] [answered]
Are you planing to add support for cuda 12.6 ?

**Comment by Titus-von-Koeller** [comment]
cc @matthewdouglas

**Reply by matthewdouglas** [reply, marked as answer]
@eljandoubi I've been testing with 12.6 builds for a little while now and things do seem stable, so it is likely that we can provide wheels for that in a near future release. In the meantime, [building from source](https://huggingface.co/docs/bitsandbytes/main/en/compiling) with 12.6 should work.
